### PR TITLE
Timezones loaded from moment instead of database

### LINKED ISSUE #681: enhance loading meter timezone
### Describe the bug

Each time the meters page loads, the timezone of each meter is fetched separately, taking a lot of requests. This makes the rate limit in PR #676 high, which is very risky. Optimization is needed to reduce the number of requests

### To Reproduce

1. Click the "Meters" button
2. Look at the Docker console window, you will see the timezones requests starting with "GET /api/timezones"

### Expected behavior

The number of timezones requests is reduced.

### PR DESCRIPTION
**Pull request by Jake-Bodin**
(Please include a summary of the change and which issue is touched on. Please also include relevant motivation and context.)

Fixes #681 

(In general, OED likes to have at least one issue associated with each pull request. Replace [issue] with the OED GitHub issue number. In the preview you will see an issue description if you hover over that number. You can create one yourself before doing this pull request. This is where details are normally given on what is being addressed. Note you should not use the word "Fixes" if it does not completely address the issue since the issue would automatically be closed on merging the pull request. In that case use "Partly Addresses #[issue].)

## Type of change

(Check the ones that apply by placing an "x" instead of the space in the [ ] so it becomes [x])

- [ X] Note merging this changes the database configuration.
- [ ] This change requires a documentation update

## Checklist

(Note what you have done by placing an "x" instead of the space in the [ ] so it becomes [x]. It is hoped you do all of them.)

- [ X] I have followed the [OED pull request](https://openenergydashboard.github.io/developer/pr.html) ideas
- [X ] I have removed text in ( ) from the issue request
- [X ] You acknowledge that every person contributing to this work has signed the [OED Contributing License Agreement](https://openenergydashboard.github.io/developer/cla.html) and each author is listed in the Description section.

## Limitations

(Describe any issues that remain or work that should still be done.)

### CODE REVIEW

File: src/client/app/components/TimeZoneSelect.tsx [hunk 1]
```diff
@@ -4,42 +4,35 @@
...
+		});
+	};
+	const resetTimeZone = [{ value: null, label: translate('timezone.no') }];
+	const options = [...resetTimeZone, ...getTimeZones()];
 
 	const handleChange = (selectedOption: TimeZoneOption) => {
 		handleClick(selectedOption.value);
 	};
 
 	return (options !== null ?
```
Comment (huss) [inline src/client/app/components/TimeZoneSelect.tsx]: The need to check for null options was a result of the way the data was requested from the database. With it now being gotten from moment this cannot happen. I removed the conditional and used:
	return <Select isClearable={false} value={options.filter(({ value }) => value === current)} options={options} onChange={handleChange} />;
It seems to work fine. I think this change should be moade if @Jake-Bodin does not find any issues with is.

**Review by huss — CHANGES_REQUESTED** [review]
Thanks to @Jake-Bodin for addressing this issue. Testing found it worked fine. I have one small comment to address to clean up code.

File: src/client/app/components/TimeZoneSelect.tsx (hunk 1 above)
Comment (Jake-Bodin) [reply src/client/app/components/TimeZoneSelect.tsx]: I don't see any changes made from my original code, I don't feel I know enough to know if the new timezones are properly effecting the meters but i don't see any changes or problems.

Comment (huss) [reply src/client/app/components/TimeZoneSelect.tsx]: Sorry this was not clear. The comment had the desired code. I made this change and push a commit to put it into the PR. This PR should now be ready to merge.

**Review by huss — APPROVED** [review]

**Pull request merged**